apps/web/playwright/reschedule.e2e.ts

```diff
@@ -416,6 +418,112 @@ test.describe("Reschedule Tests", async () => {
       }
     });
   });
+
+  test.describe("Guest availability-awareness reschedule by host", () => {
+    test("host reschedule with full non-cal.com user guests shouldn't trigger guests availability awareness", async ({
+      page,
+      users,
+      bookings,
+    }) => {
+      const host = await users.create();
+      const eventType = host.eventTypes[0];
+      const nonCalUserAttendees = Array.from({ length: 10 }, (_, i) => ({
+        email: `attendee-${i}@example.com`,
+        name: `Attendee Example ${i}`,
+        timeZone: "Europe/London",
+      }));
+      const booking = await bookings.create(host.id, host.username, eventType.id, {
+        attendees: {
+          createMany: {
+            data: nonCalUserAttendees,
+          },
+        },
+      });
+
+      await page.goto(`/reschedule/${booking.uid}`);
+
+      // Let current month dates fully render.
+      await page.click('[data-testid="incrementMonth"]');
+      // Waiting for full month increment
+      await page.locator('[data-testid="day"][data-disabled="false"]').nth(0).click();
+      // Expect normal full availability which 32 time slots
+      await expect(page.locator('[data-testid="time"]')).toHaveCount(32);
+    });
+    test("host reschedule with full cal.com user guests should trigger guests availability awareness", async ({
+      page,
+      users,
+      bookings,
+    }) => {
+      const host = await users.create();
+      const guests = await Promise.all(
+        Array.from({ length: 10 }, (_, i) => users.create({ username: `guest-${i}` }))
```

Comment:
No need to add these many guests. Add 2 guests with separate availability timings and having some bookings for them.